docs/04-FunctionalTests.md

```diff
@@ -250,10 +267,43 @@ error_level: "E_ALL & ~E_STRICT & ~E_DEPRECATED"
 
 `error_level` can also be set globally in `codeception.yml` file.
 
+## Accessing external URL's with Guzzle
```

Comment:
In advance: The code I posted worked for me! Is that Symfony-specific?

So I'm not sure what you mean:

1. It's wrong because it doesn't work. My answer: See above :-)
1. It's wrong because you shouldn't do it this way. My answer: Then how should it be done?

My main point: I see a need for accessing external url's in functional tests; people are certainly asking this question. So there should be an answer. If this pull request is the wrong answer, then let's change it to the right! :-)